docs/contribute.rst

```diff
@@ -247,6 +247,26 @@ squash the changes into a single commit. Use multiple commits to
 group relevant code changes if the pull request is too large for a
 single commit.
 
+Consider pasting the following checklist (or selected items which
```

Comment:
How about adding "Fixes: #<issue ID>" to the PR's description so as to associate PR with the corresponding issue?

Reply:
Good idea! Added in b57bad79.